libs/components/forms/src/lib/modules/toggle-switch/toggle-switch.component.ts

```diff
@@ -54,7 +54,8 @@ export class SkyToggleSwitchComponent
   /**
    * The ARIA label for the toggle switch. This sets the toggle switch's `aria-label`
    * attribute to provide a text equivalent for screen readers [to support accessibility](https://developer.blackbaud.com/skyux/learn/accessibility).
-   * If the `sky-toggle-switch-label` component displays a visible label, do not use this property.
+   * Use a context-sensitive label, such as "Annual fundraiser is active." Context is especially important when multiple toggle switches are in close proximity.
+   * If the `sky-toggle-switch-label` component displays a visible label, use this property if extra context is needed due to that context only being available visibly.
```

Comment:
```suggestion
   * If the `sky-toggle-switch-label` component displays a visible label, this property is only necessary when that label requires extra context.
```
I don't feel like the second half of this sentence was very clear. I think we need "only" in there to convey that using this property when there is a visible label is an exception to the rule. And I don't understand what we mean by "context only being available visibly" given that it's presumably the label that should provide context and screen readers will pick it up. So I rewrote to add "only" and work around the bit at the end. Do you think this rewrite works?